build.py

```diff
@@ -244,25 +209,30 @@ def create_tags(id, biome, winter_biomes: list, bare_winter_biomes: list, snowy_
     non_spring = union(vanilla, summer_list, fall_list, winter_list, melting_list)
     any = union(vanilla, summer_list, fall_list, winter_list, melting_list, spring_list)
 
-    write_tag(f'non_summer/{id}', non_summer)
-    write_tag(f'non_fall/{id}', non_fall)
-    write_tag(f'non_winter/{id}', non_winter)
-    write_tag(f'non_spring/{id}', non_spring)
-    write_tag(f'any/{id}', any)
+    write_tag(f"non_summer/{biome_id}", non_summer)
+    write_tag(f"non_fall/{biome_id}", non_fall)
+    write_tag(f"non_winter/{biome_id}", non_winter)
+    write_tag(f"non_spring/{biome_id}", non_spring)
+    write_tag(f"any/{biome_id}", any)
+
 
-def int_to_rgb(color: int) -> tuple:
-    r = (color >> 16) & 0xff
-    g = (color >> 8) & 0xff
-    b = color & 0xff
+def int_to_rgb(color: int) -> tuple[float, ...]:
+    r = (color >> 16) & 0xFF
+    g = (color >> 8) & 0xFF
+    b = color & 0xFF
     return r / 255.0, g / 255.0, b / 255.0
 
-def rgb_to_int(r: int, g: int, b: int) -> int:
-    r = r * 255
-    g = g * 255
-    b = b * 255
-    return ((int(r) & 0xff) << 16) | ((int(g) & 0xff) << 8) | (int(b) & 0xff)
 
-def calculate_color(downfall: float, temperature: float, colormap: numpy.typing.NDArray) -> int:
+def rgb_to_int(r: float, g: float, b: float) -> int:
+    r = int(r) * 255
+    g = int(g) * 255
+    b = int(b) * 255
```

Comment:
So this is the line with the intent change, and it prompted me to have a look at the actual diffs:

```diff
diff -r post/seasons/worldgen/biome/winter_snowy/windswept_hills.json pre/seasons/worldgen/biome/winter_snowy/windswept_hills.json
18c18
<     "sky_color": 0,
---
>     "sky_color": 10925528,
21c21
<     "grass_color": 255,
---
>     "grass_color": 16056063,
```

And--yup, I can see what's going on and that the *type hint* was the part that needed to be fixed. Please hold.